### PR TITLE
2.0.0 stable

### PR DESCRIPTION
**Pull request by michalchudziak**
Summary:
---------

Stable 2.0.0 release. Updated readme, bumped RN and RN Cli, resolved lodash vulnerability, and added missing debug keystore.

Test Plan:
----------

N/A

### CODE REVIEW

File: example/ios/Podfile.lock [hunk 1]
```diff
@@ -97,7 +97,7 @@ DEPENDENCIES:
   - React-jsi (from `../../node_modules/react-native/ReactCommon/jsi`)
   - React-jsiexecutor (from `../../node_modules/react-native/ReactCommon/jsiexecutor`)
   - React-jsinspector (from `../../node_modules/react-native/ReactCommon/jsinspector`)
-  - react-native-slider (from `/Users/thymikee/Projects/oss/react-native-slider`)
```
Comment (michalchudziak) [inline example/ios/Podfile.lock]: @thymikee 👀

File: README.md [hunk 2]
```diff
@@ -6,85 +6,23 @@ React Native component used to select a single value from a range of values.
...
-
-![xcode-add](https://facebook.github.io/react-native/docs/assets/AddToLibraries.png)
+## React Native Compatibility
+To use this library you need to ensure you are using the correct version of React Native.
 
-### Link `libRNCSlider.a` binary with libraries
+| `@react-native-community/slider` version | Required React Native Version |
+| ---------------------------------------- | ----------------------------- |
+| `2.x.x`                                  | `>= 0.60`                     |
+| `1.x.x`                                  | `<= 0.59`                     |
```
Comment (thymikee) [inline README.md]: maybe add a link to 1.x branch which will have relevant docs on linking?

File: package.json [hunk 3]
```diff
@@ -50,13 +50,13 @@
...
     "react": "16.8.6",
-    "react-native": "0.60.0-rc.3",
+    "react-native": "0.60.4",
     "react-test-renderer": "16.8.6"
   },
   "resolutions": {
     "handlebars": "4.1.2",
     "js-yaml": "3.13.1",
-    "@react-native-community/cli": "^2.0.0-rc.5"
+    "lodash": "4.17.13"
```
Comment (thymikee) [inline package.json]: All of these resolutions are likely not necessary. Make sure they're latest possible in the lock file and then hopefully remove

Comment (michalchudziak) [reply package.json]: I don't really want Github to scream at me each time I visit this repo :<

File: example/ios/Podfile.lock (hunk 1 above)
Comment (thymikee) [reply example/ios/Podfile.lock]: Yep, that's expected :)

File: package.json (hunk 3 above)
Comment (thymikee) [reply package.json]: Remove yarn.lock and add again, should make them gone :D but if not, keep the resolutions field

Comment (michalchudziak) [reply package.json]: Not sure until we merge, but yeah, I'll try that ^^

**Pull request merged**